src/app/src/components/EmbeddedMapFieldsConfig.jsx

```diff
@@ -271,10 +331,24 @@ function EmbeddedMapFieldsConfig({
                                             />
                                         }
                                         label="Select All"
+                                        style={{ width: '312px' }}
                                         classes={{
                                             label: classes.listText,
                                         }}
                                     />
+                                    <Tooltip title="Choose which visible fields to include in text search for your map. Facility name, address, and OAR ID will always be included.">
+                                        <FormLabel
+                                            classes={{
+                                                label: classes.listText,
+                                            }}
+                                            style={{ color: 'rgb(0,0,0)' }}
+                                        >
+                                            Searchable{' '}
+                                            <i
+                                                className={`${classes.icon} fas fa-fw fa-info`}
```

Comment:
If we already use `fa-info` as the tooltip icon elsewhere in the app, then this is fine. But if not, can we change it to `fa-question-circle`

Reply:
This is the first place we've added a tooltip icon, so we can definitely use `fa-question-circle`